pkg/assembler/graphql/schema/certifypkg.graphql

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright 2023 The GUAC Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: This is experimental and might change in the future!
+
+# Defines a GraphQL schema for the CertifyPkg. It contains a list of packages that are similar
+# along with the justification, origin and collector.
+"""
+CertifyPkg is an attestation that represents when a package objects are similar
+
+Packages - list of package objects
+Justification - string value representing why the packages are similar
+Origin - where this attestation was generated from (based on which document)
+Collector - the GUAC collector that collected the document that generated this attestation
+
+"""
+type CertifyPkg {
```

Comment:
nit: can we add maybe for these evidence nodes inline comments to indicate which ones are external subjects/objects and properties?

Reply:
Yes I can do this for them all once I have them merged in.

Reply:
Created https://github.com/guacsec/guac/issues/458 to track